guillotina/traversal.py

```diff
@@ -446,10 +446,14 @@ async def real_resolve(self, request: IRequest) -> Optional[MatchInfo]:
                         }
                     )
 
-        if not view and len(tail) > 0:
-            # we should have a view in this case because we are matching routes
-            await notify(TraversalViewMissEvent(request, tail))
-            raise HTTPNotFound(content={"reason": "object and/or route not found"})
+        if view is None and len(tail) > 0:
+            # Try arbitrary "path" in the path
+            view_name = tail[0] + "?"
+            view = query_multi_adapter((resource, request), method, name=view_name)
```

Comment:
Why we need this here ?

Reply:
the adapter has been registered with the name `@endpoint?` instead of `@endpoint///` pattern, just a convention that we can modify to be more explicit.